brave-tests/src/main/java/brave/test/propagation/PropagationTest.java

```diff
@@ -180,8 +180,9 @@ public MapEntry() {
       Map<Object, String> map = new LinkedHashMap<>();
       injector.inject(ctx, map);
 
-      assertThat(extractor.extract(map).context())
```

Comment:
assertj got weird and leaked data which I didn't feel like hunting down. This is a classloader test